tools/segments/src/api/query.graphql.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2025 Adobe
+ * All Rights Reserved.
+ */
+
+async function executeGraphQlQuery(query, config) {
+  const headers = {
+    'Content-Type': 'application/json',
+    'x-api-key': config['commerce.headers.cs.x-api-key'],
+    'Magento-Customer-Group': config['commerce.headers.cs.Magento-Customer-Group'],
+    'Magento-Environment-Id': config['commerce.headers.cs.Magento-Environment-Id'],
```

Comment:
Not all headers are neccessary for Adobe Commerce

Reply:
Removed unnecessary headers.